libindy/src/api/anoncreds.rs

```diff
@@ -1154,6 +1154,7 @@ pub  extern fn indy_prover_close_credentials_search(command_handle: i32,
 /// Annoncreds*
 /// Common*
 /// Wallet*
+#[deprecated(since="1.6.1", note="Please use indy_prover_search_credentials_for_proof_req instead!")]
```

Comment:
Please add the same for `indy_prover_get_credentials` too

Reply:
please review.